js/deleteTask.js

```diff
@@ -0,0 +1,6 @@
+export default function removeTodo(element, taskList) {
+  const index = taskList.findIndex((item) => item.id == element.id);
```

Comment:
the implementation currently has a strong dependency on a DOM element with an `id` property and a reference to the `parentNode`. It's making it hard to mock the function arguments(dependencies) for the unit test. 

let's refactor the function to take the todo `id` as the last argument.

```suggestion
export default function removeTodo(element, taskList, id) {
  const index = taskList.findIndex((item) => item.id == id);
```